Detect new browsers and apps - Tint Browser, hola!, RedReader

sanchezzzhak opened the pull request.

sanchezzzhak committed:
feat(client) detect new browsers for clienthints: Tint Browser (org.tint), Zirco Browser (org.zirco)
feat(client) detect browser for clienthint: 360 Browser (com.qihoo.browser, com.qihoo.haosou, com.qihoo.padbrowser)
feat(client) detect new apps: Reddit is fun
feat(client)!: rename 360 Phone Browser to 360 Browser

sanchezzzhak committed:
feat(client) detect new app: Daum

sgiehl commented on Parser/Client/Browser.php:
```diff
@@ -68,7 +68,6 @@ class Browser extends AbstractClientParser
         'DW' => 'Dark Web Browser',
         '1B' => '115 Browser',
         '2B' => '2345 Browser',
-        '36' => '360 Phone Browser',
```
wondering if it wouldn't be better to keep the record and maybe give it the same name as below, so the short code is still valid and usable.

sanchezzzhak replied:
I returned the changes.

sanchezzzhak committed:
feat(client) detect app for clienthints: RedReader, Sina Weibo
feat(client) detect new browser for clienthints: Sony Small Browser
feat(client) detect browser for clienthints: Skyfire

sanchezzzhak committed:
fix: phpcs

sanchezzzhak committed:
fix: tests

sanchezzzhak committed:
feat(client) detect browser for clienthints: hola! Browser

sanchezzzhak committed:
feat(client) detect apps: Google Search App, Baidu Input

sanchezzzhak committed:
Merge branch 'master' into master
